tcs-api/src/main/java/com/transformuk/hee/tis/tcs/api/enumeration/PostEsrEventStatus.java

```diff
@@ -6,5 +6,6 @@
 public enum PostEsrEventStatus {
 
   RECONCILED,
-  DELETED
+  DELETED,
+  RECONCILED_EXP
```

Comment:
Based on conversations, I believe that `EXP` is short for expired rather than an unusual bit of ESR terminology like APC.
Would `EXPIRED` or `EXPIRED_RECONCILIATION` be a suitable name?  I also like `INVALIDATED` slightly more than `OBSOLETE` as a more common way of saying `DEPRICATED`.  If `REPLACED` isn't too confusing, that would be more consistent with "plain english".